Foundation/NSNumberFormatter.swift

```diff
@@ -180,9 +179,17 @@ open class NumberFormatter : Formatter {
         
         set {
             switch newValue {
-            case .none, .ordinal, .spellOut:
+            case .none, .ordinal:
                 _usesSignificantDigits = false
-                
+
+            case .spellOut:
+                // FIXME: Locale identifier necessary to make unum work; cheat by using a "default"
+                // Should determine proper current locale identifier
+                if locale.identifier.length == 0 {
+                    _locale = Locale(identifier: "en_US")
```

Comment:
Thanks for the tip!

A bit apprehensive about changing something "highly optimized" :-)
Is the update suitable?
The no-locale-ID might've only been a Mac issue.

And if there's ever time, I would love to hear the story behind CF's whitespace situation... it would warm my heart if, say, the computer it was on was low on hard drive space!